feat: declare support for nil (#276)
nix-community/vscode-nix-ide: README.md

  ![](./images/docs/md-embed-nix.png)

> * Full editing support with any nix [LSP](https://microsoft.github.io/language-server-protocol/) implementation such as [rnix-lsp](https://github.com/nix-community/rnix-lsp) or [nil](https://github.com/oxalica/nil).

Reviewer: Instead of saying any LSP with confidence, I think we can add some `generally` or such words? 

and mention that it has been tested to work with the following LSP servers ?

Author: Yeah, good point. I've updated the wording; let me know if that hits the spot or not.

Reviewer: This looks good. Just a small nit though, please check.

Author: Works for me :+1: suggested change applied.

  ![](./images/docs/md-embed-nix.png)

> * Full editing support when using a language server. Generally, any nix [LSP](https://microsoft.github.io/language-server-protocol/) implementation should work but only the following have been tested:

Reviewer: ```suggestion
* Full editing support when using a language server. Generally, any nix [LSP](https://microsoft.github.io/language-server-protocol/) implementation should work.  
The following are tested so far:
```
